android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -292,34 +293,14 @@ public void setSupportMultipleWindows(RNCWebView view, boolean enabled){
 
   @ReactProp(name = "webViewKey")
   public void setWebViewKey(RNCWebView view, String webViewKey) {
-    Map<String, WebView> internalWebViewMap = RNCWebViewMapManager.INSTANCE.getInternalWebViewMap();
-    Map<String, RNCWebView> rncWebViewMap = RNCWebViewMapManager.INSTANCE.getRncWebViewMap();
+    view.setWebViewKey(webViewKey);
 
-    // If there is an existing RNCWebView that has an internal webview, re-attach it to this view
-    if (rncWebViewMap.containsKey(webViewKey)) {
-      RNCWebView existingView = rncWebViewMap.get(webViewKey);
-      InternalWebView existingWebView = existingView.detachWebView();
-      view.attachWebView(existingWebView);
-
-      // The chrome client was originally setup on instance creation but might be pointing to the wrong webview
-      // so it's reset here.
-      // Not entirely sure why there is a single instance of the webchrome client for all webviews?
-      setupWebChromeClient((ThemedReactContext) existingWebView.getContext(), existingWebView);
```

Comment:
I'm seeing this deleted but not added back anywhere. Do we still need to call `setupWebChromeClient` anywhere?

Reply:
Would it have to get called again in `onAttachedToWindow` when the webViewKey is non-null?